packages/HelpAndCancel/Microsoft.Bot.Components.HelpAndCancel.nuspec

```diff
@@ -8,7 +8,7 @@
         <copyright>© Microsoft Corporation. All rights reserved.</copyright>
         <licenseUrl>https://github.com/Microsoft/BotBuilder/blob/master/LICENSE</licenseUrl>
         <requireLicenseAcceptance>true</requireLicenseAcceptance>
-        <projectUrl>https://github.com/Microsoft/botframework-components</projectUrl>
+        <projectUrl>https://github.com/Microsoft/botframework-components/tree/main/packages/HelpAndCancel</projectUrl>
         <releaseNotes>First version of HelpAndCancel package</releaseNotes>
         <description>Dialogs to provide help and cancel utterance handling</description>
         <tags>msbot-component msbot-content conversationalcore helpintent cancelintent preview</tags>
```

Comment:
Can we use a consistent style for tags? Some are doing semi-colon delimited, this doing space. I don't mind either or, just would be good to have one.

Reply:
That _would_ be good, but all examples I have seen show that .nuspec uses `<tags>` with spaces between them, while .csproj uses `<PackageTags>` with semi-colons...

Reply:
![image](https://user-images.githubusercontent.com/43043272/114763001-31c90b80-9d17-11eb-8dde-0def42c8d8b1.png)
![image](https://user-images.githubusercontent.com/43043272/114763085-473e3580-9d17-11eb-8833-e9acab83ff6c.png)